tokio-test/CHANGELOG.md

```diff
@@ -1,3 +1,11 @@
+# 0.4.2 (March 20, 2021)
+
+### Fixed
+
+- test: Make `Mock` both `Send` and `Sync` ([#3594])
```

Comment:
This seems included in 0.4.1...
https://github.com/tokio-rs/tokio/commits/master/tokio-test

Reply:
Ah yeah I see. I guess someone made a release without a git tag for it.